pyof/foundation/basic_types.py

```diff
@@ -169,6 +169,31 @@ def unpack(self, buff, offset=0):
         self._value = unpacked_data.decode('ascii').rstrip('\0')
 
 
+class IPAddress(GenericType):
+    """Defines a IP address."""
+
+    _fmt = "!I"
+    netmask = UBInt32()
+
+    def __init__(self, address=None, netmask=32):
+        """The constructor takes the parameters below.
+
+        Args:
+            address (UBInt32): Ip address. Defaults to '0.0.0.0'.
+            netmask(UBInt32): Subnet Mask. Defaults to 32.
```

Comment:
It is not clear to me if we should go with a netmask attribute as integer or on the 'IPAddress' format (XXX.XXX.XXX.XXX). @beraldoleal , what do you think?